### ISSUE TITLE
BE: francais abbreviations aren’t French

### ISSUE THREAD

**Issue by georgd** [Juris-M/legal-resource-registry#17]
Dutch and French are official languages in Belgium with regions either officially French or Dutch except for Brussels that’s bilingual (plus a region where German has a certain status). `abbrev:francais` looks like it should be used for French abbreviations but it doesn‘t, the file lists Dutch abbreviations under those keys.

**Comment by mvwestendorp** [comment]
There is an overall bigger issue with the abbreviations for Belgium in that currently for French, every court is a variant of the (dutch) abbreviation and `Anvers` and the Dutch file has the court right after the name `MechelenArbrb`. From what I see a refactoring here https://github.com/Juris-M/jurism-abbreviations/commit/684a094d3b63fc70c06e60b423f6bacafe149f94 is the cause. I was actually planning to create an new issue for that.

I have contributed the lists for Belgium via @fbennett at an earlier stage and still have these files (see attachement).  Are those still usable (given the new data) or is it better to just start correcting the `juris-be-desc.json`? If so, I am willing to make those corrections.

[auto-be.json.zip](https://github.com/Juris-M/legal-resource-registry/files/5256892/auto-be.json.zip)

**Comment by georgd** [comment]
> There is an overall bigger issue with the abbreviations for Belgium in that currently for French, every court is a variant of the (dutch) abbreviation and `Anvers` and the Dutch file has the court right after the name `MechelenArbrb`. 

I thought so already :)

> From what I see a refactoring here [Juris-M/jurism-abbreviations@684a094](https://github.com/Juris-M/jurism-abbreviations/commit/684a094d3b63fc70c06e60b423f6bacafe149f94) is the cause. I was actually planning to create an new issue for that.
> 
> I have contributed the lists for Belgium via @fbennett at an earlier stage and still have these files (see attachement). Are those still usable (given the new data) or is it better to just start correcting the `juris-be-desc.json`? If so, I am willing to make those corrections and also provide the correct French abbreviations while at it

If anyone could, it would be Frank. But given that already two refactorings have passed since, and the capabilities have greatly evolved, I think it’s better to directly work on `juris-be-desc.json`.

**Comment by georgd** [comment]
Looking more closely through the old files, I have the impression that the French file wasn’t complete then, as well (but maybe I just don’t fully understand the Belgian Tower of Babel): e.g., I’d not expect `Vred. Liège` or `Assisen Namur` in there. – So, one more reason for correcting the current `desc` file directly.

**Comment by mvwestendorp** [comment]
Yes you're right about the French file. I am not French speaking so I looked primarily at the Dutch file. 
Another reason is that French isn't that relevant for the stylesheet that I use/support ([V&A](https://github.com/Juris-M/jm-styles)), which is exclusively Dutch.

The examples you give are part Dutch abbreviation (`Vred.` ~ `Vrederechter`) with a French city (`Liege` ~ `Luik`) so it is related to the issue you describe in the first post. I have to check with someone who is French speaking for this.

I remember that Frank at one point had could go both ways between those files, but maybe I was mistaken. Probably the refactoring does not allow that anymore so I will just start working on transferring the information to the `desc` file unless Frank has a turnkey solution :)

**Comment by fbennett** [comment]
@mvwestendorp The new format should offer all the locale flexibility that you need. The main (non-variant) entries are language-agnostic, so if some courts are known exclusively in one language, and some in another, languages can be mixed at top-level. You can then tweak things for alternative citation output (drawn from name/abbrev and ABBREV) or alternative locales in the UI (drawn from name exclusively).

Files in repo have all been converted to the new format. If you have an old-format file to hand with preferred content, I can help convert that to the new format with a script (the script is in the repo, but requires some configuration to make it go).

**Comment by fbennett** [comment]
@mvwestendorp (Oh -- I think I may have misread your message? If the problem is that the better Dutch content is in an abbreviation file, then yes, that will need to be carried across to ``juris-desc``.)

**Comment by mvwestendorp** [comment]
@fbennett  Thanks for the clarification. No problem for carrying it over. It all needs to be checked anyway.

I am working on it and will report back later.
